deep-learning/_posts/2018-07-05-Ship-wake-detection.md

```diff
@@ -0,0 +1,66 @@
+
+A new conjugate gradient training method as applied to back-propagation learning is superior to the conventional steepest-descent method.
+
+## Preprocessing
+
+They use concepts of plane-wave detection in the Fourier space to develop the algorithm.
+
+![](/deep-learning/images/Ship_wake_detection/1991_preprocessing.png)
+
+"Line detection" is the Fourier transform of a function `f(x,y)` with the Fourier power spectrum by `|F|^2 = FF*`.
+
+"feature vector" is the normalization of the signal to its sum.
+
+"Feature list" is a pair of numbers, one for each tile, consisting of the maximum probability and its angle.
+
+![](/deep-learning/images/Ship_wake_detection/1991_samples.png)
+
+## Network
+
+![](/deep-learning/images/Ship_wake_detection/1991_network.png)
+
+Summary:
+1. A starting $$\vec{x}_{\,0}$$ point is selected by initializing the weights randomly, uniformly distributed between -0.5 and +0.5. The gradient $$\vec{g}_{\,0}$$ is computed at this point and an initial search direction vector $$\vec{d}_{\,0}=-\vec{g}_{\,0}$$ is selected.
+
+2. The constant ak which minimizes $$f(\vec{x}_{\,k}+\alpha_{\,k}\vec{d}_{\,k})$$ is computed by line search. The weight vector is updated to the new point: $$\vec{x}_{\,k+1}=\vec{x}_{\,k}+\alpha_{\,k}\vec{d}_{\,k}$$.
+
+3. The termination criteria are evaluated at the new point. If the error at this point is acceptable or if the maximum number of function and gradient calculations has been reached, the algorithm terminates.
+
+4. A new direction vector $$\vec{d}_{\,k+1}$$ is computed. If $$ k+1 $$ is an integral multiple of $$N$$ (the dimension of $$\vec{x}$$), then $$\vec{d}_{\,k+1} = -\vec{g}_{\,k+1}$$. Otherwise, $$\vec{d}_{\,k+1} = -\vec{g}_{\,k+1}+\beta_{k}\vec{d}_{\,k}$$, where  is computed according to the method selected by the user: Fletcher-Reeves, Polak-Ribikre, or Hestenes-Stiefel.
+
+5. $$k$$ is replaced by $$k + 1$$ , and the algorithm continues as before in Step 2.
+
+To augment the number of images, they created a simulator to generate synthetic feature lists, since they have only a limited number of SAR images. They generated feature lists for wakes oriented at 0” and 180”, with arm angles varying between 0” and 20” (a 0 arm angle simulated a wake in which one of the arms was obscured), over a range of variances. The training set consists of 232 simulated feature lists, of which half are ambient. **The network was trained using the simulated feature lists only** and tested on real data. The feature lists were presented to the network in the form of one-dimensional data, interleaving probability with an angle.
+
+## Results
+
+They using five of these methods in our neural network implementation: Fletcher-Reeves, Polak-Ribi&re, Hestenes-Stiefel, Shanno, and steepest descent. The learning parameters, which govern the convergence of the line search, are 0.1, 0.3, 0.5, 0.7, and 0.9.
```

Comment:
Review this sentence... "They **use** five of these methods in **their** neural network [...]" is probably what you mean?